code/script.js

```diff
@@ -29,19 +49,252 @@ const showMessage = (message, sender) => {
+        const choiceBtn = document.querySelectorAll(".choice-btn")
+        for (let i = 0; i < choiceBtn.length; i++) {
+          choiceBtn[i].addEventListener("click", () => nxtQuestion(choiceBtn[i].textContent))
+        }
+      }
+    })
+    document.getElementById("action").addEventListener("click", () => {
+      setTimeout (() => showMessage("I will go for action.", "user"), 1000)
+      setTimeout (() => showMessage("Great choice! Go ahead and pick the best one.", "bot"), 2000)
+      setTimeout(() => {
+        whatMovie()
+      }, 3000)
+      let whatMovie = () => {
+        inputWrapper.innerHTML =  `
+          <button class="choice-btn" id="honest-thief">Honest Thief</button>
+          <button class="choice-btn" id="tenet">Tenet</button>
+          <button class="choice-btn" id="bloodshot">Bloodshot</button>`
+        //Looping through each button and checking when it is clicked
+        const choiceBtn = document.querySelectorAll(".choice-btn")
+        for (let i = 0; i < choiceBtn.length; i++) {
+        choiceBtn[i].addEventListener("click", () => nxtQuestion(choiceBtn[i].textContent))
+        }
+      }
+    })
+    document.getElementById("comedy").addEventListener("click", () => {
+      setTimeout (() => showMessage("I will go for comedy.", "user"), 1000)
+      setTimeout (() => showMessage("Great choice! Go ahead and pick the best one.", "bot"), 2000)
+      setTimeout(() => {
+        whatMovie()
+      }, 3000)
+      let whatMovie = () => {
+        inputWrapper.innerHTML = `
+          <button class="choice-btn" id="babyteeth">Babyteeth</button>
+          <button class="choice-btn" id="downhill">Downhill</button>
+          <button class="choice-btn" id="friendsgiving">Friendsgiving</button>`
+        //Looping through each button and checking when it is clicked
+        const choiceBtn = document.querySelectorAll(".choice-btn")
+        for (let i = 0; i < choiceBtn.length; i++) {
+          choiceBtn[i].addEventListener("click", () => nxtQuestion(choiceBtn[i].textContent))
+        }
+      }
+    })
+    document.getElementById("horror").addEventListener("click", () => {
+      setTimeout (() => showMessage("I will go for horror.", "user"), 1000)
+      setTimeout (() => showMessage("Great choice! Go ahead and pick the best one.", "bot"), 2000)
+      setTimeout(() => {
+        whatMovie()
+      }, 3000)
+      let whatMovie = () => {
+        inputWrapper.innerHTML = `
+          <button class="choice-btn" id="antebellum">Antebellum</button>
+          <button class="choice-btn" id="brahms">Brahms: The Boy 2</button>
+          <button class="choice-btn" id="come-play">Come Play</button> `
+        //Looping through each button and checking when it is clicked
+        const choiceBtn = document.querySelectorAll(".choice-btn")
+        for (let i = 0; i < choiceBtn.length; i++) {
+          choiceBtn[i].addEventListener("click", () => nxtQuestion(choiceBtn[i].textContent))
+        }
+      }
+    })
+};
```

Comment:
This is a very long function with a fairly large amount of repeated code. 🤓 I think it is an opportunity to experiment and try to cut it into smaller reusable functions 😸 If you wanna try you always can ask for more tips on how to do it 😺